src/main/drivers/timer.h

```diff
@@ -82,6 +82,7 @@ typedef enum {
     TIM_USE_TRANSPONDER    = 0x20,
     TIM_USE_BEEPER         = 0x40,
     TIM_USE_CAMERA_CONTROL = 0x80,
+    TIM_USE_BBSHOT         = 0x100, // BB-Dshot pacer
```

Comment:
`TIM_USE_BITBANGING_DSHOT_PACER`, and then get rid of the comment? ;-)

Reply:
That flag I could eliminate after moving to a separate timerHardware array for bb